zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -481,19 +481,24 @@ protected Object jobRun() throws Throwable {
       try {
         InterpreterContext.set(context);
 
+        InterpreterResult result = null;
+
         // Open the interpreter instance prior to calling interpret().
         // This is necessary because the earliest we can register a hook
         // is from within the open() method.
         LazyOpenInterpreter lazy = (LazyOpenInterpreter) interpreter;
         if (!lazy.isOpen()) {
           lazy.open();
+          result = lazy.executePrecode(context);
```

Comment:
@Leemoonsoo 
works same as the previous jdbc precode.
case:
   user: some_user
   JDBC interpreter (per user)
   1. there is precode `CREATE table IF NOT EXISTS tbl_#{user}`
   2. user executes `insert into tbm_some_user ...`
      - execute `precode`
      - execute paragraph text
  2. user executes `select *from tbm_some_user`
     - execute paragraph text

Spark   
1. there is precode `var someConst = 12 ...`
2. User executes `var result = 1 + someConst`
   - execute `precode`
   - execute paragraph text
3. User executes var 
```
result2 = 2 + someConts
print(result2)
```
- execute only paragraph text